src/database/mod.rs

```diff
@@ -179,6 +347,7 @@ mod tests {
         let stop_words = HashSet::new();
 
         let meilidb_path = dir.path().join("meilidb.mdb");
+        let meilidb_index_name = "meilidb";
```

Comment:
It could have been better to use "default" but it is a test, don't change that :)